pkg/Cpanel/Security/Advisor/Assessors/Kernel.pm

```diff
@@ -90,6 +114,9 @@ sub _check_for_kernel_version {
             ],
         );
     }
+    elsif ( $kc_kernelversion ) {
```

Comment:
FYI: If you move this `elsif` block above the preceding `elsif` block, the code simplifies a bit.